rdkit/Chem/Draw/qtCanvas.py

```diff
@@ -10,7 +9,10 @@
 #
 
 from rdkit.Chem.Draw.canvasbase import CanvasBase
-from PySide import QtGui, QtCore
+try:
+  from PySide import QtGui, QtCore
+except ImportError:
+  from PyQt5 import QtGui, QtCore
```

Comment:
I only have PyQt5 in my environment. The test works and adding `img.save('deleteme.png')` gives me a png with the rendered image. So, works as far as I can see.